test/vagrant-service-manager/installer_test.rb

```diff
@@ -29,7 +29,7 @@ module ServiceManager
           @installer.instance_variable_get('@type').must_equal @type
           @installer.instance_variable_get('@machine').must_equal @machine
           @installer.instance_variable_get('@env').must_equal @machine.env
-          @installer.instance_variable_get('@box_version').must_equal 'adb'
+          @installer.instance_variable_get('@options').must_equal({ type: @type }.merge(@options))
```

Comment:
Even the test seems odd with this merge thing you are doing. Btw, I don't think these tests really do much. They are pretty much self fulfilling. I'd go so far as to say let's remove them